### ISSUE TITLE
Movie Editor - Enhance Filtering

### ISSUE THREAD

**Issue by sienar1** [Radarr/Radarr#3707]
**Is your feature request related to a problem? Please describe.**
Large movie collections are very unwieldy to manage in Radarr. The movie editor is very hard to work with when the library has thousands of movies in it. For instance, even with existing filters I can see missing movies but not released AND missing movies.

**Describe the solution you'd like**
Make the columns themselves filterable and add more library fields as columns to the movie editor. Make visibility of columns themselves togglable, don't need them all visibile all the time. Additional columns could be custom formats, date movie added, date movie file added, media info, file size, separate column for root folder (apart from the full path so you can filter movies by which root folder they're in), In Cinemas date (would allow to SORT AND filter by release date, not just release state), if list import tags get implemented then list import tags, etc

**Describe alternatives you've considered**
Sorting by existing columns is ok, but still very difficult to bulk operations on a subset of movies that might be hundreds of movies out of thousands.

**Additional context**
Basically make the movie editor as close to working in Excel as possible but with the ability make bulk changes to the library.

**Comment by henkin67** [comment]
Hello,
I agree: I have a 800 movies library and it's the hell to use the movies editor.
Clearly there are missing filtering capabilities.
This feature request should be great to have.
Regards.

**Comment by Qstick** [comment]
Same as #1621

**Comment by sienar1** [comment]
I don’t feel that this is a duplication of #1621z My original draft actually referenced #1621, but what I’m suggesting is much more in depth and so I didn’t think it was correct to hijack and massively expand the scope of #1621. Is that what I should do or can we reopen this and consider it as an alternative to #1621?
